.travis.yml

```diff
@@ -118,8 +119,6 @@ jobs:
     os: linux
     dist: focal
     compiler: clang
-    python:
-      - "3.9"
```

Comment:
this setting was ignored anyways (requires `language: python`, not `language: c`)